generator-mediator-js/app/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "enablePrompts":true,
+  "mediatorName":"Yeoman Generated Mediator",
+  "mediatorDesc":"Brief Description",
+  "mediatorAuthor":"Jembi Health Systems NPC",
+  "mediatorMaintainer":"Name Surname <email@exchange.com> (https://github.com/githubusername)",
+  "configPort":7000,
+  "defaultChannelPath":"channelPath",
+  "mediatorRouteHost":"localhost",
+  "mediatorRoutePath":"routePath",
```

Comment:
I think that instead of setting a routePath the mediator should rather accept all requests. Like the following:

```
  app.all(mediatorConfig.endpoints[0].path, (req, res) => {
    console.log(`Processing ${req.method} request on ${mediatorConfig.endpoints[0].path}`)
```